CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - [Web] Add views for handlers
 
 ### Changed
-- [Web] Updated embedded web assets from `9d91d7f` ... `275386a`
+- [Web] Vendored `sensu/web` @ SHA `0e6c347`
+- [Web] Updated embedded web assets from `9d91d7f` ... `0e6c347`
```

Comment:
The release branch has already been cut, you'll want to put this under the `Unreleased` heading.